server/main.go

```diff
@@ -39,21 +39,24 @@ var (
 	port       = flag.String("port", ":12345", "Port on which to run the server.")
 )
 
+// Version is the release version number.
```

Comment:
please clarify which version this refers to (the API?) and a clear note to update this with each branch.

Reply:
Done